App/.env

```diff
@@ -0,0 +1,11 @@
+AAD_CLOUD_INSTANCE=Enter_the_Cloud_Instance_Id_Here # cloud instance string should end with a trailing slash
+AAD_TENANT_INFO=Enter_the_Tenant_Info_Here
+AAD_CLIENT_ID=Enter_the_Application_Id_Here
+AAD_CLIENT_SECRET=Enter_the_Client_Secret_Here
+
+AAD_REDIRECT_URI=http://localhost:3000/auth/redirect
+AAD_POST_LOGOUT_REDIRECT_URI=http://localhost:3000
+
+GRAPH_API_ENDPOINT=Enter_the_Graph_Endpoint_Here # graph api endpoint string should end with a trailing slash
+
+EXPRESS_SESSION_SECRET=Enter_the_Express_Session_Secret_Here
```

Comment:
Do we need to add guidance where to get this?

Reply:
Any hard to guess value would do here, but I'll make sure to add an explanation in README